client/homebrew/pages/archivePage/archivePage.jsx

```diff
@@ -164,6 +164,16 @@ const ArchivePage = createClass({
         );
     },
 
+    validateInput: function (e) {
+        const textInput = e.target
+        const submitButton = document.getElementById('searchButton');
+        if (textInput.value.length > 3) {
+            submitButton.disabled = false;
+        } else {
+            submitButton.disabled = true;
+        }
+    },
+
     renderForm: function () {
```

Comment:
Regarding the "form":

I'm noticing now that the Search button doesn't actually do anything for me when I click on it.  I can trigger a search by hitting Enter in the Title input, but not by clicking on the button (and it isn't disabled).  No errors in console.

I think this has to do with the `onClick` function, but I checked out your branch to work on my local machine and just couldn't figure out what was up.  I did manage to fix it by converting the whole "form" div to an actual HTML `<form>` element, and changing the Search button into a `<button type='submit' onSubmit='....'>` and removing the onClick attribute (the same functions go into onSubmit).  It's hard to be sure it works on local, though, as it seems I can't actually perform the search.

Reply:
Damn it, changed the class to an id last night, forgot to change the appropiate selectors for both CSS and js. Should be using refs anyway, but will fix this now to keep testing.

Reply:
actually that can't be it, not sure what the problem is here.